src/h5web/visualizations/heatmap/hooks.ts

```diff
@@ -14,10 +14,10 @@ import { TextureWorker } from './worker';
 import { useVisProps } from '../../dataset-visualizer/VisProvider';
 
 export function useData(): number[][] {
-  const { rawValues, rawDims } = useVisProps();
+  const { visValues: mappedValues, rawDims } = useVisProps();
 
   if (rawDims.length === 2) {
-    return rawValues;
+    return mappedValues;
   }
 
   throw new Error('Data not supported by HeatmapVis');
```

Comment:
I think you can just return `visValues` as is for now, without any check/throw. You did remove the throw in the line vis' hook and I think it makes sense.